util/state.js

```diff
@@ -208,6 +208,14 @@ return 0;
     return await this.redis.hset(this.crawlSizeKey, this.uid, size);
   }
 
+  async isCrawlStopped() {
+    return await this.redis.get("crawl-stop") === "1";
+  }
+
+  async stopCrawl() {
+    await this.redis.set("crawl-stop", "1");
+  }
```

Comment:
I don't see a place where this method is getting called - is the idea that we'll be setting this in redis from Browsertrix Cloud but we just want a method in the crawler to be able to do the same if needed?

Reply:
Yep, that's gonna be next PR is to add to the operator to set this. maybe should even remove it from here, but left it in for complteness, can add a comment though.